Jenkinsfile

```diff
@@ -38,7 +38,7 @@
  */
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value="pipeline-lib@your_branch") _
+//@Library(value="pipeline-lib@ampittma-codespell") _
```

Comment:
So this isn't needed any more due to the alternate approach in daos-stack/code_review#21?  Are you sure that having it here is not interfering (as in making the code_review changes work where without it they will fail) with this PR?

Reply:
Correct, this is a "try things and see what works" PR, it's still commented out here, all that's changed is the branch name in the comment so I'm sure it won't have impacted other changes.

Reply:
Ah yes.  I didn't notice the `//` at the start.  I guess that line should be removed before landing.

Reply:
This PR should be abandoned now, and we can proceed with https://github.com/daos-stack/daos/pull/2643

All that's needed in that one is the code-review one to land, and codespell to be added to the pip install line, which I think it already is.